src/main/resources/db/migration/V1_95__create_sequence_data_file_no.sql

```diff
@@ -0,0 +1,8 @@
+create sequence juror_mod.data_file_no
```

Comment:
please can this be renamed to print_file_count instead of data_file_no?

Reply:
Updated